app/engines/bing_formatted_query.rb

```diff
@@ -1,4 +1,4 @@
-class AzureFormattedQuery < FormattedQuery
+class BingFormattedQuery < FormattedQuery
   DEFAULT_DOMAIN_SCOPE = 'site:gov OR site:mil'.freeze
```

Comment:
Is this replacing AzureFormattedQuery? I still see it in coverage/index.html line 43909.

Reply:
Sounds like that's from a  previous test run. If you nuke your `coverage` directory and re-run the specs, you should not see those old lines in `coverage/index.html`. (Searching with `git grep` should also ignore the `coverage` directory, as it's listed in `.gitignore`.)